### ISSUE TITLE
Use form label elements correctly throughout the site

### ISSUE THREAD

**Issue by NomeQ** [Trillium-Solutions/mendocino-wp-theme#5]
All form elements should have a correctly associated `<label>`. Marking this as medium because there may be more form elements than I am aware of.

https://webaim.org/techniques/forms/controls

**Comment by NomeQ** [comment]
Homepage form still does not have <label> elements for each <input>
- Remove weird "table" used for styling
- Ensure that Start, End, etc are in <label> elements and correctly attached to their related inputs

**Comment by NomeQ** [comment]
Oh wow. Thought I had hard refreshed. I had not. Apologies, I see now that this is there.
